cpp/ql/src/semmle/code/cpp/security/Overflow.qll

```diff
@@ -13,19 +13,32 @@ predicate guardedAbs(Operation e, Expr use) {
   )
 }
 
+pragma[inline]
+private predicate stmtDominates(Stmt dominator, Stmt dominated) {
+  // In same block
+  exists(BasicBlock block, int dominatorIndex, int dominatedIndex |
+    block.getNode(dominatorIndex) = dominator and
+    block.getNode(dominatedIndex) = dominated and
```

Comment:
This may result in "get all cfg nodes in the same basic block as this Stmt", which may be large.  This potential problem can be reduced by adding a helper predicate that is `getNode` reduced to just those nodes that are `instanceof Stmt`.

Reply:
Thanks! Fixed.